modules/core/04-channel/keeper/handshake.go

```diff
@@ -282,7 +282,7 @@ func (k Keeper) WriteOpenAckChannel(
 	channel.Counterparty.ChannelId = counterpartyChannelID
 	k.SetChannel(ctx, portID, channelID, channel)
 
-	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", channel.State.String(), "new-state", "OPEN")
+	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", channel.State.String(), "new-state", types.OPEN.String())
```

Comment:
Looks like previous state here will be `OPEN` too here by using `channel.State.String()`, as its assigned above on line 280. This is incorrect, I would expect it to be `INIT`.